scripts/ci/run-jenkins.sh

```diff
@@ -89,8 +89,15 @@ if [[ ${CI_TAGS} == *'product-sdks-ios'* ]];
 	   echo "DISABLE_ANDROID=1" > sdks/Make.config
 	   echo "DISABLE_WASM=1" >> sdks/Make.config
 	   export device_test_suites="Mono.Runtime.Tests System.Core"
-	   ${TESTCMD} --label=runtimes --timeout=60m --fatal make -j4 -C sdks/builds package-ios-sim64 package-ios-target64
-	   ${TESTCMD} --label=cross --timeout=60m --fatal make -j4 -C sdks/builds package-ios-cross64 package-ios-cross32
+
+	   export sim_targets="sim64 sim32 simtv simwatch"
+	   export runtime_targets="target64 target32 targettv targetwatch"
+	   export cross_targets="cross64 cross32 crosswatch"
+
+	   for arch in ${sim_targets}; do ${TESTCMD} --label=build-sim-runtime-${arch} --timeout=20m --fatal make -j4 -C sdks/builds package-ios-${arch}; done
```

Comment:
This turns off parallel builds.

Reply:
hmm, I would argue that this is preferable so we can easily spot breakages. but I don't have strong opinions on that.

Reply:
fixed